numpy/core/tests/test_memmap.py

```diff
@@ -81,7 +81,10 @@ def test_path(self):
         tmpname = mktemp('', 'mmap', dir=self.tempdir)
```

Comment:
This still isn't teardown safe against failure, but I think that can wait until we use a higher level function as a context manager, maybe `NamedTemporaryFile`.